client/components/marketing-survey/cancel-jetpack-form/jetpack-cancellation-offer.tsx

```diff
@@ -0,0 +1,137 @@
+import { PLAN_ANNUAL_PERIOD, PLAN_MONTHLY_PERIOD } from '@automattic/calypso-products';
+import { Button } from '@automattic/components';
+import formatCurrency from '@automattic/format-currency';
+import { localizeUrl } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+import * as React from 'react';
+import { useCallback, useMemo } from 'react';
+import FormattedHeader from 'calypso/components/formatted-header';
+import InlineSupportLink from 'calypso/components/inline-support-link';
+import JetpackLogo from 'calypso/components/jetpack-logo';
+import { CancellationOffer } from 'calypso/state/cancellation-offers/types';
+import type { Purchase } from 'calypso/lib/purchases/types';
+
+interface Props {
+	purchase: Purchase;
+	siteId: number;
+	offer: CancellationOffer;
+	percentDiscount: number;
+}
+
+const JetpackCancellationOffer: React.FC< Props > = ( props ) => {
+	const { offer, purchase, percentDiscount } = props;
+	const translate = useTranslate();
+
+	const { interval, singularInterval } = useMemo( () => {
+		const periods = offer.discountedPeriods;
+		const billingInterval = purchase.billPeriodDays;
+		let interval;
+		let singularInterval;
+
+		switch ( billingInterval ) {
+			case PLAN_MONTHLY_PERIOD:
+				interval = translate( '%(count)s months', { args: { count: periods } } );
+				singularInterval = translate( 'month' );
+				break;
+
+			case PLAN_ANNUAL_PERIOD:
+				if ( periods > 1 ) {
+					interval = translate( '%(count)s years', { args: { count: periods } } );
+				}
+				interval = translate( 'year' );
+				singularInterval = translate( 'year' );
+				break;
+
+			default:
+				if ( periods > 1 ) {
+					interval = translate( '%(count)s renewals', { args: { count: periods } } );
+				}
+				interval = translate( 'renewal' );
+				singularInterval = translate( 'renewal' );
+				break;
+		}
+
+		return { interval, singularInterval };
+	}, [ purchase, offer ] );
+
+	const onClickAccept = useCallback( () => {
+		return;
+	}, [] );
+
+	return (
+		<>
+			<FormattedHeader
+				headerText={ translate( 'Thanks for your feedback' ) }
+				subHeaderText={ translate(
+					'We’d love to help make Jetpack work for you. Would the special offer below interest you?'
+				) }
+				align="center"
+				isSecondary={ true }
```

Comment:
```suggestion
				isSecondary
```

Reply:
Not sure if this is a best practice or more preference, what do you think?
Changing this one and the other instances mentioned to be more the more concise version. In some ways I think the verbosity of typing out the default value is nice/ more readable, but I can see how it is redundant :)